addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init.d/odoo

```diff
@@ -20,6 +20,7 @@ DESC=odoo
 CONFIG=/home/pi/odoo/addons/point_of_sale/tools/posbox/configuration/odoo.conf
 LOGFILE=/var/log/odoo/odoo-server.log
 PIDFILE=/var/run/${NAME}.pid
+MODULES=$(ls /home/pi/odoo/addons/ -m | tr -d ' ')
```

Comment:
Any reason to use `/home/pi` instead of `$HOME` ?

Reply:
Most environment variables are not available in these services